### PR TITLE
fix(DB): Make Sunreaver Dragonhawk not attackable in Argent Tournament Grounds

### PR DESCRIPTION
**Pull request by superstyro**
Sunreaver Dragonhawk should not attackable in Argent Tournament Grounds that is next to Fight Master Helidan Lightwing.

##### CHANGES PROPOSED:
Make Sunreaver Dragonhawk in the Argent Tournament Grounds not attackable
``` sql
UPDATE `creature_template` SET `unit_flags` = 256 WHERE (`entry` = 35984);
```
##### ISSUES ADDRESSED:
Sunreaver Dragonhawk next to Flight Master Helidan Lightwing in the Argent Tournament Grounds can be attacked with Alliance character.

##### TESTS PERFORMED:
- Added update to database
- Started server on Ubuntu 18.04
- Use Alliance character
- .go creature 202485
- Attempt to attack Sunreaver Dragonhawk

##### HOW TO TEST THE CHANGES:

 -  Alliance character
 - .go creature 202485
 - Attempt to attack Sunreaver Dragonhawk

##### KNOWN ISSUES AND TODO LIST:

- None

##### Target branch(es):
- [x] Master

 

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1571346137981297600.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1571346137981297600');
+
+UPDATE `creature_template` SET `unit_flags` = 256 WHERE (`entry` = 35984);
```
Comment (Winfidonarleyan) [inline data/sql/updates/pending_db_world/rev_1571346137981297600.sql]: In the future, there should be a new line at the end of the file

Comment (Stoabrogga) [reply data/sql/updates/pending_db_world/rev_1571346137981297600.sql]: I added a newline and removed the parentheses, they are not needed here.

**Comment by Stoabrogga** [comment]
Tested successfully, thanks!

File: data/sql/updates/pending_db_world/rev_1571346137981297600.sql (hunk 1 above)
Comment (superstyro) [reply data/sql/updates/pending_db_world/rev_1571346137981297600.sql]: Thanks for the feedback on my first PR!

**Pull request merged**